src/Components/Patient/PatientHome.tsx

```diff
@@ -618,22 +669,31 @@ export const PatientHome = (props: any) => {
                 Discharge
             </Button>
               <Dialog
+                maxWidth={'md'}
                 open={openDischargeDialog}
-                onClose={handleDischargeClose}
-              >
-                <DialogTitle id="alert-dialog-title">Authorize Patient Discharge</DialogTitle>
-                <DialogContent>
-                  <DialogContentText id="alert-dialog-description">
-                    Please confirm patient Discharge
-                </DialogContentText>
+                onClose={handleDischargeClose}>
+                <DialogTitle className="flex justify-center">
+                  Before we discharge {patientData.name}
+                </DialogTitle>
+                <DialogContent className="px-20">
+                  <FormControl component="fieldset">
+                    <FormLabel component="legend" className="flex justify-center w-full">
+                      Is the patient willing to donate blood for Plasma?
```

Comment:
let me know if the text has to be changed!

Reply:
Looks good